js/libs/builder.js

```
@@ -1,14 +1,31 @@
+/**
+ * builder.js – Fliplet Form Builder: Studio-side logic for configuring forms.
+ *
+ * Handles widget initialization, parent lookup, field migration, UI observers,
+ * and Vue instance for the builder interface.
+ *
+ * All exported functions and major logic blocks are documented with JSDoc.
+ */
+
 /* eslint-disable eqeqeq */
 var widgetId = parseInt(Fliplet.Widget.getDefaultId(), 10);
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)